python/paddle/fluid/tests/unittests/mkldnn/test_mul_int8_mkldnn_op.py

```diff
@@ -52,11 +52,11 @@ def init_data(self):
 
         # limit random range inside |-127, 127| to avoid overflow on SKL
         if self.srctype == np.int8:
-            A_data = np.random.randint(-127, 127, (2, 5)).astype(np.int8)
-        else:
-            A_data = np.random.randint(0, 127, (2, 5)).astype(np.uint8)
+            A_data = np.random.randint(-127, 127, (20, 5)).astype(np.int8)
+        elif self.srctype == np.uint8:
```

Comment:
Is there an option other than `np.int8` and `np.uint8`? If yes, please add `else`. If not, please keep `else` here.